libraries/standard/json/utest/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+include("../jsonFilePaths.cmake")
+project ("json unit test")
+cmake_minimum_required (VERSION 3.13)
+
+# ====================  Define your project name (edit) ========================
+set(project_name "json")
+
+# =====================  Create your mock here  (edit)  ========================
+# ================= Create the library under test here (edit) ==================
+
+# list the files you would like to test here
+list(APPEND real_source_files
+            ${JSON_SOURCES}
+        )
+# list the directories the module under test includes
+list(APPEND real_include_directories
+            .
```

Comment:
Are `.` and `${LOGGING_INCLUDE_DIRS}` dependencies required?

Reply:
Same for `test_include_directories`

Reply:
You are right. Forgot to remove these.